### DISCUSSION TITLE
Trouble understanding Weather Forecast API

### DISCUSSION THREAD

**Discussion by prince-of-maroc** [open-meteo/open-meteo#135] [Q&A] [answered]
Does index 0 of the arrays inside of the **hourly** object represent the _current hour_? Or does it represent 00:00 at the beginning of the _current day_ in GMT?

**Comment by patrick-zippenfenig** [comment, marked as answer]
The first index 0 is the beginning of the current day

There is also an array with `time` that contain the timestamp for each index
